docs/clusterdefinition.md

```diff
@@ -471,16 +471,18 @@ We consider `kubeletConfig`, `controllerManagerConfig`, `apiServerConfig`, and `
 | distro                       | no                                        | Select Master(s) Operating System (Linux only). Currently supported values are: `ubuntu`, `aks` and `coreos` (CoreOS support is currently experimental). Defaults to `aks` if undefined. `aks` is a custom image based on `ubuntu` that comes with pre-installed software necessary for Kubernetes deployments. Currently supported OS and orchestrator configurations -- `ubuntu` and `aks`: DCOS, Docker Swarm, Kubernetes; `RHEL`: OpenShift; `coreos`: Kubernetes. [Example of CoreOS Master with CoreOS Agents](../examples/coreos/kubernetes-coreos.json) |
 | customFiles                  | no                                        | The custom files to be provisioned to the master nodes. Defined as an array of json objects with each defined as `"source":"absolute-local-path", "dest":"absolute-path-on-masternodes"`.[See examples](../examples/customfiles)                                                                                                                                                                                           |
+| availabilityProfile          | no                                                                   | Supported values are `AvailabilitySet` (default) and `VirtualMachineScaleSets` (requires Kubernetes clusters version 1.10+ and agent pool availabilityProfile must also be `VirtualMachineScaleSets`).                                                                                                                                                                                                                                                                                                                                                                                             |
+| agentVnetSubnetId                 | only required when using custom VNET and when MasterProfile is using `VirtualMachineScaleSets`                                         | Specifies the Id of an alternate VNET subnet for the agent nodes. The subnet id must specify a valid VNET ID owned by the same subscription. ([bring your own VNET examples](../examples/vnet)). When MasterProfile is using `VirtualMachineScaleSets`, this value should be the subnetId of the agent subnet.                                                                                                                                                                                                                                                |
```

Comment:
Sort of the same question here, what does "the agent nodes" mean? what happens when there are several agent pools? Also what is the need for having the agent subnet in the master profile?